### ISSUE TITLE
wrongfully detects `no-unregistered-classes` on object usage within `cx`

### ISSUE THREAD

**Issue by beckerei** [schoero/eslint-plugin-readable-tailwind#226]
Condensed example

```tsx
const status = 'some-value'

cx({
      initial: 'border-dashed border-neutral',
      'some-value': 'border-solid,
}[status])}
```

`some-value` object key is detected as classname reporting `no-unregistered-classes`.
if the actual classname on the right-hand side is invalid, it does not seem to be registered by this plugin.
